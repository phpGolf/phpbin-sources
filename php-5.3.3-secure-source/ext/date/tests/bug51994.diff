005+   int(1)
005-   int(6)
007+   int(154)
007-   int(3)
017+   int(1)
017-   int(0)
019+   array(1) {
020+     [5]=>
021+     string(27) "The parsed date was invalid"
019-   array(0) {