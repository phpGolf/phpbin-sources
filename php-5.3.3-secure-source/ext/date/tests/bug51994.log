
---- EXPECTED OUTPUT
array(12) {
  ["year"]=>
  int(2010)
  ["month"]=>
  int(6)
  ["day"]=>
  int(3)
  ["hour"]=>
  bool(false)
  ["minute"]=>
  bool(false)
  ["second"]=>
  bool(false)
  ["fraction"]=>
  bool(false)
  ["warning_count"]=>
  int(0)
  ["warnings"]=>
  array(0) {
  }
  ["error_count"]=>
  int(0)
  ["errors"]=>
  array(0) {
  }
  ["is_localtime"]=>
  bool(false)
}
---- ACTUAL OUTPUT
array(12) {
  ["year"]=>
  int(2010)
  ["month"]=>
  int(1)
  ["day"]=>
  int(154)
  ["hour"]=>
  bool(false)
  ["minute"]=>
  bool(false)
  ["second"]=>
  bool(false)
  ["fraction"]=>
  bool(false)
  ["warning_count"]=>
  int(1)
  ["warnings"]=>
  array(1) {
    [5]=>
    string(27) "The parsed date was invalid"
  }
  ["error_count"]=>
  int(0)
  ["errors"]=>
  array(0) {
  }
  ["is_localtime"]=>
  bool(false)
}
---- FAILED
