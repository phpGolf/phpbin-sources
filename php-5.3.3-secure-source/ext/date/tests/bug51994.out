array(12) {
  ["year"]=>
  int(2010)
  ["month"]=>
  int(1)
  ["day"]=>
  int(154)
  ["hour"]=>
  bool(false)
  ["minute"]=>
  bool(false)
  ["second"]=>
  bool(false)
  ["fraction"]=>
  bool(false)
  ["warning_count"]=>
  int(1)
  ["warnings"]=>
  array(1) {
    [5]=>
    string(27) "The parsed date was invalid"
  }
  ["error_count"]=>
  int(0)
  ["errors"]=>
  array(0) {
  }
  ["is_localtime"]=>
  bool(false)
}